Sources/ObjC/RSRSSParser.m

```diff
@@ -300,7 +300,18 @@ - (NSString *)urlString:(NSString *)s {
 	}
 
 	if (!self.link) {
-		//TODO: get feed URL and use that to resolve URL.*/
+        NSURL *feedURL = [NSURL URLWithString:self.urlString];
+        if (!feedURL) {
+            return s;
+        }
+        NSURL *baseURL = [NSURL URLWithString:[NSString stringWithFormat:@"%@://%@", feedURL.scheme, feedURL.host]];
+        if (!baseURL) {
+            return s;
+        }
+        NSURL *resolvedURL = [NSURL URLWithString:s relativeToURL:baseURL];
```

Comment:
Do we need the `baseURL` step? My thinking is that we get the resolvedURL using the feedURL, something like:

`NSURL *resolvedURL = [NSURL URLWithString:s relativeToURL:feedURL];`

Reply:
I apologize, but I'm not well-acquainted with the NSURL API